pyanaconda/screen_access.py

```diff
@@ -93,12 +93,15 @@ def write_out_config_file(self, config_path=None):
 
         with self._lock:
             new_config_file = not os.path.exists(config_path)
-            with open(config_path, "wt") as f:
-                if new_config_file:
-                    # we are creating a new file, so add a header that it was created by Anaconda,
-                    # including its version number
-                    f.write(self._get_new_config_header())
-                self._config.write(f)
+            try:
+                with open(config_path, "wt") as f:
+                    if new_config_file:
+                        # we are creating a new file, so add a header that it was created by Anaconda,
+                        # including its version number
+                        f.write(self._get_new_config_header())
+                    self._config.write(f)
+            except OSError:
+                log.exception("Can't write user interaction config file.")
```

Comment:
Yep - `log.exception()` logs the message you give it and then the full traceback from the exception caught by the except block. From [the docs](https://docs.python.org/3.5/library/logging.html#logging.Logger.exception):

```
Logger.exception(msg, *args, **kwargs)

    Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(). Exception info is added to the logging message. This method should only be called from an exception handler.
```